Source/Evolve/WarpXEvolve.cpp

```diff
@@ -165,6 +165,7 @@ WarpX::Evolve (int numsteps)
         if (do_electrostatic != ElectrostaticSolverAlgo::None)
         {
             const bool skip_deposition = true;
+            if (warpx_py_particleinjection) warpx_py_particleinjection();
```

Comment:
LGTM, only called otherwise in `OneStep_nosub` which is not taken for the ES solver

Reply:
Thanks for the change. Another option is to put the call to `warpx_py_particleinjection` at line 162, before the `if` blocks (and remove it from `OneStep_nosub`). This would make the option available to all of the variants of the step.

Reply:
I like that as well :+1:

Reply:
I agree with @dpgrote 's comment. @nruof Could you change this PR accordingly?

Reply:
Changes are up.  warpx_py_particleinjection is no longer in Onestep_nosub and is at line 162